source/manual/data-gov-uk-troubleshooting.html.md

```diff
@@ -0,0 +1,53 @@
+---
+owner_slack: "#govuk-platform-health"
+title: Troubleshooting data.gov.uk
+section: data.gov.uk
+layout: manual_layout
+parent: "/manual.html"
+last_reviewed_on: 2018-08-15
+review_in: 6 months
+---
+[find]: apps/datagovuk_find
+[publish]: apps/datagovuk_publish
+[ckan]: apps/ckanext-datagovuk
+
+## Different number of datasets in [CKAN] to [Find]
+
+Determine the number of datasets in CKAN using the API.
+
+```
+https://data.gov.uk/api/3/action/package_search
```

Comment:
Bear in mind this is not the query the workers uses to schedule its work.

Reply:
Updated this too.  Oddly the endpoints return different counts.